### DISCUSSION TITLE
gpg not on my windows 10 to verify SO v2

### DISCUSSION THREAD

**Discussion by iqworks** [Security-Onion-Solutions/securityonion#4843] [Unsupported Versions]
HI, following the SO v2.3 installation from "https://github.com/Security-Onion-Solutions/securityonion/blob/master/VERIFY_ISO.md", i went to verify the iso. but when i ran this line, i get this "C:\WINDOWS\system32>gpg --verify securityonion-2.3.60-FBPIPELINE.iso.sig securityonion-2.3.60-FBPIPELINE.iso
'gpg' is not recognized as an internal or external command,
operable program or batch file.". 
  i finally downloaded kleopatra, but i cannot find the detached signature for the SO-2.3.60-fbpipeline text. 
  i tried to download GnuPG, but couldnt down load it for free and when i did something seemed to go wrong. pngtools needed java, etc. 
  i wonder if i can just get the signature for the iso in text so i can use kleopatra. 
thanks for any help or advice

**Comment by iqworks** [comment]
Why was this closed? Did I author it wrong? I could use the help

 

From: Mike Reeves ***@***.***> 
Sent: Sunday, July 18, 2021 8:04 AM
To: Security-Onion-Solutions/securityonion
***@***.***>
Cc: iqworks ***@***.***>; Author ***@***.***>
Subject: Re: [Security-Onion-Solutions/securityonion] gpg not on my windows
10 to verify SO v2 (#4839)

 

Closed #4839
<https://github.com/Security-Onion-Solutions/securityonion/issues/4839> .

-
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/Security-Onion-Solutions/securityonion/issues/4839#event
-5035124135> , or unsubscribe
<https://github.com/notifications/unsubscribe-auth/ABKPV34KK72JLD757TM5BPDTY
LUOZANCNFSM5ARQSUVA> .
<https://github.com/notifications/beacon/ABKPV32XFX3Y6VUCWUTO2QDTYLUOZA5CNFS
M5ARQSUVKYY3PNVWWK3TUL52HS4DFWZEXG43VMVCXMZLOORHG65DJMZUWGYLUNFXW5KTDN5WW2ZL
OORPWSZGPAAAAAAJMDXS2O.gif> 



-- 
This email has been checked for viruses by AVG.
https://www.avg.com

**Reply by TOoSmOotH** [reply]
please use discussions for problems like this

**Reply by TOoSmOotH** [reply]
I went ahead and converted it

**Comment by iqworks** [comment]
Thanks Mike, I will. I am new to this forum.

**Comment by iqworks** [comment]
Converted it?  is this one in discussions now as well?

thanks
